### DISCUSSION TITLE
Wireguard not working with the new version anymore

### DISCUSSION THREAD

**Discussion by rkosick** [outdoorbits/little-backup-box#256] [General]
I installed the new, now stable version but wireguard is not working anymore. I've configured it so that when I backup with rsync server wireguard is used to get to the server. I checked again with an older version that is still working. The config file is stored correctly in /etc/wireguard/wg0.conf but when lbb tries to connect it stops at "Connect VPN".

**Comment by outdoorbits** [comment]
Are you using bullseye?

**Reply by rkosick** [reply]
I can dobble check but yes, I did use the recommended version bullseye.

**Reply by rkosick** [reply]
I did doubble check. It is bullseye. It was a new installation.

**Comment by outdoorbits** [comment]
Was this installation a few days old? I had an error (issues). It should be fixed now but you need a new installation or you follow the instructions of that issue. Could you access the web UI of Comitup?

**Reply by rkosick** [reply]
Yes it was. I did prepare it last Wednesday or Thursday. I can just try a new installation.

**Comment by outdoorbits** [comment]
I think you got this bug, sorry. I'm traveling this week but the actual version should work. Only VPN wireguard probably won't work - I'll test it next week.

**Reply by rkosick** [reply]
Ok no hurry. Have a nice vacation.

**Comment by outdoorbits** [comment]
Thank you!

**Reply by outdoorbits** [reply]
It's working now. Please follow the instructions at https://github.com/outdoorbits/little-backup-box/issues/255
to fix the problem or just install completely new.

**Reply by rkosick** [reply]
I installed completely new and now everything works again. Thank you so much. Topic can be closed.

**Comment by outdoorbits** [comment]
Thank you for testing!
